------------------------------------------------
SLURM Job ID: 2176794
SLURM Array Job ID: 
SLURM Array Task ID:  / 3
Running on host: ccc0288.campuscluster.illinois.edu
Node allocated: ccc[0288,0292]
Requested CPUs: 
Running Dataset: ANLI
Running Experiment: experiment-6
Sample Size: 10000 (8000 train, 1000 validation, 1000 test)
------------------------------------------------
Modules loaded.
Python environment 'IS567' activated: /sw/apps/anaconda3/2024.10/bin/python
OMP_NUM_THREADS set to 
Changed directory to /u/jywu3/scratch/IS567FP
Starting Python script for ANLI / experiment-6 (Task ID )...

libgomp: Invalid value for environment variable OMP_NUM_THREADS

libgomp: Invalid value for environment variable OMP_NUM_THREADS
Traceback (most recent call last):
  File "/scratch/jywu3/IS567FP/main.py", line 10, in <module>
    import pandas as pd
  File "/sw/apps/anaconda3/2024.10/lib/python3.12/site-packages/pandas/__init__.py", line 49, in <module>
    from pandas.core.api import (
  File "/sw/apps/anaconda3/2024.10/lib/python3.12/site-packages/pandas/core/api.py", line 28, in <module>
    from pandas.core.arrays import Categorical
  File "/sw/apps/anaconda3/2024.10/lib/python3.12/site-packages/pandas/core/arrays/__init__.py", line 1, in <module>
    from pandas.core.arrays.arrow import ArrowExtensionArray
  File "/sw/apps/anaconda3/2024.10/lib/python3.12/site-packages/pandas/core/arrays/arrow/__init__.py", line 5, in <module>
    from pandas.core.arrays.arrow.array import ArrowExtensionArray
  File "/sw/apps/anaconda3/2024.10/lib/python3.12/site-packages/pandas/core/arrays/arrow/array.py", line 50, in <module>
    from pandas.core import (
  File "/sw/apps/anaconda3/2024.10/lib/python3.12/site-packages/pandas/core/ops/__init__.py", line 8, in <module>
    from pandas.core.ops.array_ops import (
  File "/sw/apps/anaconda3/2024.10/lib/python3.12/site-packages/pandas/core/ops/array_ops.py", line 56, in <module>
    from pandas.core.computation import expressions
  File "/sw/apps/anaconda3/2024.10/lib/python3.12/site-packages/pandas/core/computation/expressions.py", line 21, in <module>
    from pandas.core.computation.check import NUMEXPR_INSTALLED
  File "/sw/apps/anaconda3/2024.10/lib/python3.12/site-packages/pandas/core/computation/check.py", line 5, in <module>
    ne = import_optional_dependency("numexpr", errors="warn")
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/sw/apps/anaconda3/2024.10/lib/python3.12/site-packages/pandas/compat/_optional.py", line 135, in import_optional_dependency
    module = importlib.import_module(name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/sw/apps/anaconda3/2024.10/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/sw/apps/anaconda3/2024.10/lib/python3.12/site-packages/numexpr/__init__.py", line 45, in <module>
    nthreads = _init_num_threads()
               ^^^^^^^^^^^^^^^^^^^
  File "/sw/apps/anaconda3/2024.10/lib/python3.12/site-packages/numexpr/utils.py", line 156, in _init_num_threads
    requested_threads = int(os.environ['OMP_NUM_THREADS'])
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: invalid literal for int() with base 10: ''
Python script finished with exit code 1 for Task ID 
------------------------------------------------
Job Task  finished.
Total Runtime: 14 seconds
------------------------------------------------
